docs/SECURITY.md

```diff
@@ -17,6 +17,9 @@ Authentication and Authorization. But to do so usually requires
 configuring your Operating System to restrict the operations that can be done.
 This is generally a good idea even if you plan on running your cluster with Auth.
 
+Meaning to say, Storm's OS level security is based on running Storm processes with proper OS account,
```

Comment:
I think this is a little hard to understand. What do you think about "Storm's OS level security relies on running Storm processes using OS accounts that have only the permissions they need. Note that workers run under the same OS account as the Supervisor daemon by default"?

Reply:
I think that's clearer. Thanks for suggestion.

Reply:
Updated.